docs/topics/user_permissions.md

```diff
@@ -0,0 +1,51 @@
+# Users and permissions
+
+
+## What user permissions are defined in IoTP?
+
+
+In IoTP there are defined Services and SubServices. SubServices exists into Services.
+Tipicaly a Service represents a smartcity and all of the SubServices represents the verticals of that smartcity.
+
+Users are created into Services. The same user name could be used across different Services to represent different Users.
+i.e. adm1 could exists in "SmartCity" Service and adm1 could be another user for "SmartGondor" Service
+
+Roles are created into Services. By default all Services created into IoT platform are created with the following Roles:
+- ServiceCustomer: Role for a normal user of the Service, with standard capabilities.
```

Comment:
ServiceCustomer role in a given service means SubServiceCustomer role in any subservice of the service? Maybe this should be clarified (either one option or the other) in the document.

(Maybe it is, but I have reach the point where it is described :)